clusterloader2/pkg/measurement/util/checker/checker_map.go

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package checker
+
+type CheckerMap map[string]Checker
+
+// NewCheckerMap creates new checker map.
+func NewCheckerMap() CheckerMap {
+	return make(map[string]Checker)
+}
+
+// StopAllCheckers stops all checkers in the map.
+func (cm CheckerMap) StopAllCheckers() {
```

Comment:
Call it dispose and also clean up the map.

Reply:
Done.